classes/sota: no need to add wic when sota is enabled

ricardosalveti opened the pull request:
While most boards will indeed use wic format for images, it is not
something that is strictly required when using sota, as we do have
some BSPs that require only the ext4 image instead (e.g. tegra).

Wic should be included, when needed, at the machine config instead (or
via sota_<machine>.bbclass).

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>

pattivacek commented:
Would it make sense to add it to the respective `sota_*.bbclass` files, then?

Makes sense to me at any rate, though.

ricardosalveti commented:
> Would it make sense to add it to the respective `sota_*.bbclass` files, then?

Will double check if that would cause wic to be missing in any of the current classes.

ricardosalveti commented:
Added to the ones that had classes and which I couldn't find wic being defined. Also removed the :remove line from qemu as that is not required anymore.

pattivacek reviewed (approved):
Great, thank you!
